wpimath/CMakeLists.txt

```diff
@@ -77,17 +75,16 @@ if(WITH_JAVA)
     file(GLOB JACKSON_JARS "${WPILIB_BINARY_DIR}/wpiutil/thirdparty/jackson/*.jar")
     file(GLOB QUICKBUF_JAR ${WPILIB_BINARY_DIR}/wpiutil/thirdparty/quickbuf/*.jar)
 
-    set(CMAKE_JAVA_INCLUDE_PATH wpimath.jar ${EJML_JARS} ${JACKSON_JARS} ${QUICKBUF_JAR})
-
     set(CMAKE_JNI_TARGET true)
 
     file(GLOB_RECURSE JAVA_SOURCES src/main/java/*.java src/generated/main/java/*.java)
 
     add_jar(
         wpimath_jar
         ${JAVA_SOURCES}
-        INCLUDE_JARS ${EJML_JARS} wpiutil_jar wpiunits_jar
+        INCLUDE_JARS wpiutil_jar wpiunits_jar ${EJML_JARS} ${JACKSON_JARS} ${QUICKBUF_JAR}
```

Comment:
I thought INCLUDE_JARS and CMAKE_JAVA_INCLUDE_PATH did two different things, where INCLUDE_JARS actually embeds the full jar contents into the resulting jar, and JAVA_INCLUDE_PATH does not.  Do we really want to embed copies of jackson and quickbuf into the wpiutil jar?

Reply:
`INCLUDE_JARS` does not include JAR contents into the resulting JAR. [It just puts the listed JARs on the classpath for compilation.](https://cmake.org/cmake/help/latest/module/UseJava.html#command:add_jar) `CMAKE_JAVA_INCLUDE_PATH` does the same thing, but we used it in weird ways, so I just merged all usages into `INCLUDE_JARS`.